dkist/net/tests/test_attr_walker.py

```diff
@@ -53,6 +99,60 @@ def test_walker_single(all_attrs_classes, api_param_names):
     assert not set(api_param_names[all_attrs_classes]).difference(params[0].keys())
 
 
+def test_boundingbox_containing(boundingbox_param):
```

Comment:
Could we parametrize these tests with a `(kwarg, query_param)` key? as the only things which are changing between the three tests are the input and the expected strings, seems silly to write out the same test three times for that?

If you haven't user parametrize before I am talking about this: https://docs.pytest.org/en/latest/example/parametrize.html

Reply:
Yep it does seem repetitive to type out the same test 3x. I did think about this, but the tests are already parametrized with boundingbox_param. Can you double parametrize? i.e. have one boundingbox_param and also some expected string param?